app/admin/news_article.rb

```diff
@@ -0,0 +1,51 @@
+if defined?(ActiveAdmin)
+  ActiveAdmin.register News::Article, as: 'News' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.